docs/source/api_reference/transforms.rst

```diff
@@ -135,14 +135,16 @@ Scaling transforms:
    MinMaxScalerTransform
    MaxAbsScalerTransform
 
-Functional transforms:
+Functional transforms and their utilities:
 
 .. autosummary::
    :toctree: api/
    :template: class.rst
 
    LambdaTransform
    AddConstTransform
+   math.BinaryOperationTransform
```

Comment:
You should add `BinaryOperationTransform` (without `math.`).

Reply:
I think there should be `BinaryOperationTransform` and `math.BinaryOperator`.

Reply:
It also doesn't render anything for `BinaryOperator`:
![image](https://github.com/etna-team/etna/assets/142778107/4e9b3fee-639c-4f5d-a035-2a8c4f0c6587)